kore/app/repl/Main.hs

```diff
@@ -171,9 +185,14 @@ mainWithOptions
                 { SMT.timeOut = smtTimeOut
                 , SMT.preludeFile = smtPrelude
                 }
-    SMT.runSMT smtConfig
-        $ evalSimplifier emptyLogger
-        $ proveWithRepl indexedModule specDefIndexedModule initialScript
+    if replMode == RunScript && (unReplScript replScript) == Nothing
```

Comment:
I think this check should be moved in `parseKoreReplOptions`, since it is technically a parse error.